### PR TITLE
News on March 25, 2020

### PR DESCRIPTION
**Pull request by ybiquitous**
See also:

- https://github.com/sider/runners/blob/master/CHANGELOG.md#0220
- https://github.com/sider/devon_rex/blob/2.16.0/CHANGELOG.md#2160

### CODE REVIEW

File: docs/news/2020.md [hunk 1]
```diff
@@ -9,6 +9,43 @@ hide_title: true
...
+
+We are happy to announce that [remark-lint](../tools/markdown/remark-lint.md) is newly supported! 🎉
+
+This tool can detect possible mistakes in Markdown files, so it will surely help your code-review.
+It is disabled by default, so if you want to use, then please enable it in your repository settings.
+
+### New Reek options
+
+In this release, we have added the `target` and `config` options for [Reek](../tools/ruby/reek.md).
+If you need it, please configure them via your `sider.yml`.
```
Comment (yykamei) [inline docs/news/2020.md]: :thinking: In my opinion, this line is redundant. Or, it's better that there are explanations about `target` and `config` options. My suggestion is to just remove the line:

```suggestion
```

Comment (ybiquitous) [reply docs/news/2020.md]: Surely 👍

**Comment by ybiquitous** [comment]
can you review again?

**Comment by ybiquitous** [comment]
Thanks!

**Pull request merged**